docs/commerce/payment/payment_api.md

```diff
@@ -0,0 +1,68 @@
+---
+description: Use PHP API to manage payments in Commerce. You can create, update and delete payments.
+edition: commerce
+---
+
+# Payment API
+
+!!! tip "Payment REST API"
+
+    To learn how to manage payments with the REST API, see the [REST API reference](../../api/rest_api/rest_api_reference/rest_api_reference.html#payment).
+
+To get payments and manage them, use the `Ibexa\Contracts\Payment\PaymentServiceInterface` interface.
+
+From the developer's perspective, payments are referenced with a UUID identifier. 
```

Comment:
UUID is used by default to generate payment identifier. It's possible to provide custom identifier in Payment{Create/Update}Struct.